tests/unit/YourCompany.spec.ts

```diff
@@ -86,7 +90,8 @@ describe('YourCompany in a BEN correction', () => {
 })
 
 describe('YourCompany in a SP alteration', () => {
-  const store = getVuexStore()
+  setActivePinia(createPinia())
+  const store = useStore()
```

Comment:
Any idea if each describe block just uses 1 global store (in this file), or are they all separate instances?

Reply:
Might be shared, if you want a fresh store:
https://pinia.vuejs.org/cookbook/testing.html

![image](https://user-images.githubusercontent.com/3484109/231281431-8066ff03-14a7-457c-81ba-94ec356d7e1f.png)

Reply:
Good to know, thanks.